release-notes.txt

```diff
@@ -0,0 +1,38 @@
+Release v0.7.1 release notes.
+
+New features:
+    Add alert manager, define several rules for alert. Operator will receive email notification if cluster has some problem.
+    More examples and scenario: NLP training, E2E vertical, Spark. etc.
+
+Improvements:
+    Optimize boot speed of web portal in #936
+    Documentation: README restructure: more friendly to new users. Also with miscellaneous doc refinement in REST server, zookeeper and exporter.
```

Comment:
Documentation:
- Restructured and refined README to provide a better experiences for new users.
- Documentation improvement for REST Server, Zookeeper and exporter.